app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -39,6 +39,7 @@ $govuk-cookie-banner-background: govuk-colour("light-grey", "grey-4");
 
 .gem-c-cookie-banner__confirmation-message {
   margin-right: govuk-spacing(4);
+  padding-top: govuk-spacing(3);
```

Comment:
I think it would be better to have the padding on the overall component parent rather than add specific padding to child elements that does the same thing - although it looks like the problem here is the DS styles rather than yours.

I would remove the top padding from `govuk-cookie-banner` and put it on `gem-c-cookie-banner` instead, which would pad that and the confirmation message as well.

Reply:
It's messy with the required overrides, but I accept your point about applying it to the parent consistently - have updated.

Reply:
Thanks, that seems better to me. Unfortunately the spacing isn't consistent, due to a child of the confirmation message having padding of its own...

![Screenshot 2021-02-17 at 17 08 12](https://user-images.githubusercontent.com/861310/108240524-d62d3800-7142-11eb-9cd8-d7190208ca0d.png)

...but to be honest at this point if these are all DS styles then I think you've already done enough to try to make this consistent and any further discussion should happen at a different level, so I'm going to approve since you've already solved the problem you set out to solve.